src/adhocracy_core/adhocracy_core/rest/__init__.py

```diff
@@ -1,4 +1,74 @@
 """Configure rest api packages."""
+from pyramid.view import view_config
+from adhocracy_core.authentication import validate_user_headers
+from adhocracy_core.caching import set_cache_header
+from adhocracy_core.rest.schemas import validate_request_data
+from adhocracy_core.rest.schemas import validate_visibility
+
+
+class api_view(view_config):  # noqa
+    """ A function\class\method decorator to configure a rest api :term:`view`.
+
+    It works like the normal :class:`pyramid.view.view_config` decorator exept
+    the following defaults are set::
+
+        renderer='json'
+        decorator = [validate_user_headers,
+                     validate_visibility,
+                     set_cache_header]
+
```

Comment:
> I'm not very familiar with this part of the code so I'll need more practice to answer your question. Let's leave it like this for now.

If you put more logic into the decorator you could write:

   @api_view()
    def get()

instead of 

@api_view(request_method='get', permission='view')
def get()